cadwyn/routing.py

```diff
@@ -53,6 +53,7 @@
 from cadwyn.structure.versions import _CADWYN_REQUEST_PARAM_NAME, _CADWYN_RESPONSE_PARAM_NAME, VersionChange
 
 _T = TypeVar("_T", bound=Callable[..., Any])
+R = TypeVar("R", bound=fastapi.routing.APIRouter)
```

Comment:
Let's use `_R` to prevent this from becoming a public interface.

Reply:
renamed